src/jit/hwintrinsicxarch.cpp

```diff
@@ -754,14 +760,98 @@ GenTree* Compiler::impSSE2Intrinsic(NamedIntrinsic        intrinsic,
             }
             break;
 
+        case NI_SSE2_ConvertScalarToVector128Double:
+        {
+            assert(sig->numArgs == 2);
+            assert(getBaseTypeOfSIMDType(sig->retTypeSigClass) == TYP_DOUBLE);
+
+            argList = info.compCompHnd->getArgNext(sig->args);
+            CorInfoType corType =
+                strip(info.compCompHnd->getArgType(sig, argList, &argClass)); // type of the second argument
+
+            baseType = JITtype2varType(corType);
+
+#ifdef _TARGET_X86_
+            if (varTypeIsLong(JITtype2varType(corType)))
+            {
+                return impUnsupportedHWIntrinsic(CORINFO_HELP_THROW_PLATFORM_NOT_SUPPORTED, method, sig, mustExpand);
+            }
+#endif // _TARGET_X86_
+
+            if (baseType == TYP_STRUCT)
+            {
+                baseType = TYP_DOUBLE;   // it is the only type passed as Vector
+                op2      = impSIMDPopStack(TYP_SIMD16);
+            }
+            else
+            {
+                op2 = impPopStack().val;
+            }
+
+            op1     = impSIMDPopStack(TYP_SIMD16);
+            retNode = gtNewSimdHWIntrinsicNode(TYP_SIMD16, op1, op2, intrinsic, baseType, simdSize);
+
+            break;
+        }
+
+        case NI_SSE2_ConvertScalarToVector128Int64:
+        case NI_SSE2_ConvertScalarToVector128UInt64:
+        {
+            assert(sig->numArgs == 1);
```

Comment:
Due to above I have had to handle above conversion intrinsics as HW_Category_Special